lib/generic/modules/metabolic_syndrome_disease.json

```diff
@@ -0,0 +1,651 @@
+                "unit": "years"
+              }
+            ]
+          },
+          "transition" : "Chance_to_Onset_Hypertension"
+        },
+        {
+          "condition" : {
+            "condition_type" : "Vital Sign",
+            "vital_sign" : "blood_glucose",
+            "operator" : ">",
+            "value" : 6.5
+          },
+          "transition" : "Onset_Diabetes"
+        },
+        {
+          "condition" : {
+            "condition_type" : "Vital Sign",
+            "vital_sign" : "blood_glucose",
+            "operator" : ">",
+            "value" : 5.7
+          },
+          "transition" : "Onset_Prediabetes"
+        },
+        {
+          "transition" : "Potential_Diabetes"
+        }
+      ]
+    },
+
+    "Chance_to_Onset_Hypertension" : {
+      "type" : "Simple",
+      "distributed_transition" : [
+        { "distribution" : 0.594, "transition" : "Onset_Hypertension" },
+        { "distribution" : 0.406, "transition" : "No_Hypertension" }
+      ],
+      "remarks" : "probability_given_diabetes: 0.594 #  (1.0==100%) http://www.cdc.gov/MMWr/preview/mmwrhtml/su6203a24.htm#Tab"
+    },
+
+    "Onset_Hypertension" : {
+      "type" : "SetAttribute",
+      "attribute" : "hypertension",
+      "value" : true,
+      "direct_transition" : "Potential_Diabetes"
+    },
+
+    "No_Hypertension" : {
+      "type" : "SetAttribute",
+      "attribute" : "hypertension",
+      "value" : false,
+      "direct_transition" : "Potential_Diabetes"
+    },
+
+
+    "Onset_Prediabetes" : {
+      "type" : "SetAttribute",
+      "attribute" : "prediabetes",
+      "value" : true,
+      "direct_transition" : "Prediabetes_Symptom_1"
+    },
+
+    "Prediabetes_Symptom_1" : {
+      "type" : "Symptom",
+      "symptom" : "Hunger",
+      "range" : { "low" : 1, "high" : 40 },
+      "direct_transition" : "Prediabetes_Symptom_2"
+    },
+
+    "Prediabetes_Symptom_2" : {
+      "type" : "Symptom",
+      "symptom" : "Fatigue",
+      "range" : { "low" : 1, "high" : 40 },
+      "direct_transition" : "Prediabetes_Symptom_3"
+    },
+
+    "Prediabetes_Symptom_3" : {
+      "type" : "Symptom",
+      "symptom" : "Blurred Vision",
+      "range" : { "low" : 1, "high" : 40 },
+      "direct_transition" : "Prediabetes_Symptom_4"
+    },
+
+    "Prediabetes_Symptom_4" : {
+      "type" : "Symptom",
+      "symptom" : "Frequent Urination",
+      "range" : { "low" : 1, "high" : 40 },
+      "direct_transition" : "Potential_Diabetes"
+    },
+
+
+    "Onset_Diabetes" : {
+      "type" : "SetAttribute",
+      "attribute" : "diabetes",
+      "value" : true,
+      "direct_transition" : "Chance_to_Onset_Hypertension_at_Diabetes_Onset"
+    },
+
+    "Chance_to_Onset_Hypertension_at_Diabetes_Onset" : {
+      "type" : "Simple",
+      "complex_transition" : [
+        {
+          "condition" : {
+            "condition_type" : "Attribute",
+            "attribute" : "hypertension",
+            "operator" : "is not nil"
```

Comment:
I think you want to transition here if hypertension is `true` otherwise you want to recalculate whether or not they have hypertension (their probability increased)